Lua/Plugins/rust.lua

```diff
@@ -0,0 +1,120 @@
+--- Highlight Keywords
+highlight("as", "reserved")
+highlight("break", "reserved")
+highlight("const", "reserved")
+highlight("continue", "reserved")
+highlight("crate", "reserved")
+highlight("else", "reserved")
+highlight("enum", "reserved")
+highlight("extern", "reserved")
+highlight("false", "reserved")
+highlight("fn", "reserved")
+highlight("for", "reserved")
+highlight("if", "reserved")
+highlight("impl", "reserved")
+highlight("in", "reserved")
+highlight("let", "reserved")
+highlight("loop", "reserved")
+highlight("match", "reserved")
+highlight("mod", "reserved")
+highlight("move", "reserved")
+highlight("mut", "reserved")
+highlight("pub", "reserved")
+highlight("ref", "reserved")
+highlight("return", "reserved")
+highlight("self", "reserved")
+highlight("Self", "reserved")
+highlight("static", "reserved")
+highlight("struct", "reserved")
+highlight("super", "reserved")
+highlight("trait", "reserved")
+highlight("true", "reserved")
+highlight("type", "reserved")
+highlight("unsafe", "reserved")
+highlight("use", "reserved")
+highlight("where", "reserved")
+highlight("while", "reserved")
+highlight("async", "reserved")
+highlight("await", "reserved")
+highlight("dyn", "reserved")
+
+--- Arithmetic and Bitwise Operators
+highlight("+", "operator")
+highlight("-", "operator")
+highlight("*", "operator")
+highlight("/", "operator")
+highlight("%", "operator")
+highlight("<<", "operator")
+highlight(">>", "operator")
+highlight("&", "operator")
+highlight("|", "operator")
+highlight("^", "operator")
+highlight("!=", "operator")
+highlight("==", "operator")
+highlight(">", "operator")
+highlight("<", "operator")
+highlight(">=", "operator")
+highlight("<=", "operator")
+highlight("&&", "operator")
+highlight("||", "operator")
+highlight("!", "operator")
+
+--- Assignment Operators
+highlight("=", "operator")
+highlight("+=", "operator")
+highlight("-=", "operator")
+highlight("*=", "operator")
+highlight("/=", "operator")
+highlight("%=", "operator")
+highlight("&=", "operator")
+highlight("|=", "operator")
+highlight("^=", "operator")
+highlight("<<=", "operator")
+highlight(">>=", "operator")
+
+--- Special Characters
+highlight("{", "special")
+highlight("}", "special")
+highlight("(", "special")
+highlight(")", "special")
+highlight("[", "special")
+highlight("]", "special")
+highlight("->", "special")
+highlight("=>", "special")
+highlight("::", "special")
+highlight(";", "special")
+highlight("_", "special")
+highlight("'", "special")
+highlight(":", "special")
+highlight(",", "special")
+
+--- Strings and Characters
+highlight_region('"', '"', "string")
+highlight_region("'", "'", "character")
+
+--- Comments
+highlight_region("//", "", "comment", true)
+highlight_region("/*", "*/", "comment", true)
```

Comment:
```suggestion
highlight_region("/*", "*/", "comment", false)
```
the 3rd parameter for `highlight_region` determines if the highlight isn't multi-line.

Reply:
4th parameter
3rd is "comment" or "string"

=3

Reply:
> 4th parameter 3rd is "comment" or "string", etc
> 
> =3

oops